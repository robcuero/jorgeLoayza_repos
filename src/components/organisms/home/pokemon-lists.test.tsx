import { fireEvent, render } from "@testing-library/react";
import { Provider } from "react-redux";
import { Home } from ".";
import { store } from "../../../store";

describe("Pokemon organism", () => {
  it("Should render Pokemon with props", async () => {
    const app = render(<Provider store={store}><Home/></Provider>)
    expect(app).toMatchSnapshot()
  });
});
