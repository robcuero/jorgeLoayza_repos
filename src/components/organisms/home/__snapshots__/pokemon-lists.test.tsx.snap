// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemon organism Should render Pokemon with props 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-gKPRtg kOYtZz"
      >
        <div
          class="body--resvert"
        >
          <div
            class="sc-bcXHqe eQbgpN"
          >
            <h1
              class="alignTitle"
            >
              Listado de Pokemon
            </h1>
            <div
              class="sc-dkrFOg cbIfJD"
            >
              <div
                class="input-wrapper"
              >
                <input
                  class="input"
                  data-testid="id-input"
                  placeholder="Buscar"
                  type="text"
                />
                <img
                  alt="Buscar"
                  class="input-icon"
                  src="search-icon.svg"
                />
              </div>
            </div>
          </div>
          <div
            class="size"
          >
            <div
              class="sc-eDvSVe fbymGp"
            >
              <div
                class="block"
              >
                <div
                  class="sc-jSUZER gxZmBK"
                  color="#76dace"
                >
                  <div
                    class="block--details"
                  >
                    <img
                      alt=""
                      class="size-img"
                      height="20"
                      src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/undefined.png"
                      width="20"
                    />
                    <div
                      class="title"
                    >
                      <div>
                        #
                      </div>
                      <div />
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="sc-hLBbgP fzowQA"
            >
              <div
                class="block"
              >
                <div
                  class="block--details"
                >
                  <div
                    class="no-pokemons"
                  >
                    <img
                      alt=""
                      height="100"
                      src="no-pokemons.png"
                      width="100"
                    />
                    <div>
                      Escoje un pokemon
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="button-wrapper"
          >
            <div>
              <div
                class="sc-gswNZR hjjnOF"
              >
                <button
                  class="button-primary disabled"
                  disabled=""
                >
                  
                  &lt; Atras
                </button>
              </div>
            </div>
            <div>
              <div
                class="sc-gswNZR hjjnOF"
              >
                <button
                  class="button-primary disabled"
                  disabled=""
                >
                  
                  Siguiente &gt;
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-gKPRtg kOYtZz"
    >
      <div
        class="body--resvert"
      >
        <div
          class="sc-bcXHqe eQbgpN"
        >
          <h1
            class="alignTitle"
          >
            Listado de Pokemon
          </h1>
          <div
            class="sc-dkrFOg cbIfJD"
          >
            <div
              class="input-wrapper"
            >
              <input
                class="input"
                data-testid="id-input"
                placeholder="Buscar"
                type="text"
              />
              <img
                alt="Buscar"
                class="input-icon"
                src="search-icon.svg"
              />
            </div>
          </div>
        </div>
        <div
          class="size"
        >
          <div
            class="sc-eDvSVe fbymGp"
          >
            <div
              class="block"
            >
              <div
                class="sc-jSUZER gxZmBK"
                color="#76dace"
              >
                <div
                  class="block--details"
                >
                  <img
                    alt=""
                    class="size-img"
                    height="20"
                    src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/undefined.png"
                    width="20"
                  />
                  <div
                    class="title"
                  >
                    <div>
                      #
                    </div>
                    <div />
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="sc-hLBbgP fzowQA"
          >
            <div
              class="block"
            >
              <div
                class="block--details"
              >
                <div
                  class="no-pokemons"
                >
                  <img
                    alt=""
                    height="100"
                    src="no-pokemons.png"
                    width="100"
                  />
                  <div>
                    Escoje un pokemon
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="button-wrapper"
        >
          <div>
            <div
              class="sc-gswNZR hjjnOF"
            >
              <button
                class="button-primary disabled"
                disabled=""
              >
                
                &lt; Atras
              </button>
            </div>
          </div>
          <div>
            <div
              class="sc-gswNZR hjjnOF"
            >
              <button
                class="button-primary disabled"
                disabled=""
              >
                
                Siguiente &gt;
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
